contracts/HybridCustody.cdc

```diff
@@ -140,7 +140,11 @@ pub contract HybridCustody {
         // addCapabilityToProxy
         // Adds a capability to a parent's managed @ProxyAccount resource. The Capability can be made public,
         // permitting anyone to borrow it.
-        pub fun addCapabilityToProxy(parent: Address, _ cap: Capability, isPublic: Bool)
+        pub fun addCapabilityToProxy(parent: Address, _ cap: Capability, isPublic: Bool) {
```

Comment:
We should probably address the inputs where cap has the `_` preface. Thoughts on removing that for consistency?

Reply:
Agreed, good catch